### ISSUE TITLE
Appdaemon not sending entity_id

### ISSUE THREAD

**Issue by khan3962** [home-assistant/appdaemon#346]
My dashboards correctly display current state from home assistant so I know they are communicating.  However if I try to control anything from the dashboard it doesn't do anything.   Upon inspection of home assistant logs I see : [homeassistant.components] homeassistant/turn_on cannot be called without entity_id

Only thing I've changed since it worked last is installed hass.io addons influxDB and grafana.

appdaemon addon v1.3.0
Home Assistant v0.73.2

**Comment by khan3962** [comment]
I finally got it working again, and have narrowed it down to the ha_key in the configuration being auto set. : 
[cont-init.d] 30-auto-password.sh: executing... 
INFO: ha_key is missing in the AppDaemon configuration, fixing...
After which the key is set in the conf file, but my HA instance has no key.  If I change the ha_url from http://hassio/homeassistant to http://172.17.0.1:8123 (which I got from the HAdashboard v2 addon which worked) and delete ha_key it works again!
